addons/mail/views/res_users_views.xml

```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0"?>
 <odoo>
     <data>
+        <record id="action_res_users_my_fullpage" model="ir.actions.act_window">
```

Comment:
Do you really have to create a whole separate action?

Can't you change the target of the existing one when needed?